galaxy/main/migrations/0090_auto_20180411_1129.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.11 on 2018-04-11 15:29
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+UPDATE_REPOSITORY = """
+UPDATE main_repository rp SET (issue_tracker_url) = (
+  SELECT DISTINCT
+  regexp_replace(regexp_replace(issue_tracker_url, 'api.', ''), '\{/number\}', '')
+  FROM main_content c
+  WHERE c.repository_id = rp.id
+  LIMIT 1
+)
+"""
+
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0089_delete_invalid_contents'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='repository',
+            name='issue_tracker_url',
+            field=models.CharField(blank=True, max_length=256,
+                                   null=True,
+                                   verbose_name=b'Issue Tracker URL'),
+        ),
+        migrations.RunSQL(
+            sql=UPDATE_REPOSITORY,
+            reverse_sql=migrations.RunSQL.noop
```

Comment:
Is this migration reversible? If no, please don't use `noop` and omit passing `reverse_sql` parameter. In this case Django will raise exception on downgrading to earlier migration.

Reply:
Done